boards.txt

```diff
@@ -23243,3 +23243,166 @@ crabik_slot_esp32_s3.menu.EraseFlash.all=Enabled
+esp32s3_powerfeather.build.flash_mode=dio
+esp32s3_powerfeather.build.boot=qio
+esp32s3_powerfeather.build.partitions=default
+esp32s3_powerfeather.build.defines=
+esp32s3_powerfeather.build.loop_core=
+esp32s3_powerfeather.build.event_core=
+esp32s3_powerfeather.build.flash_type=qio
+esp32s3_powerfeather.build.psram_type=qspi
+esp32s3_powerfeather.build.memory_type={build.flash_type}_{build.psram_type}
+
+esp32s3_powerfeather.menu.LoopCore.1=Core 1
+esp32s3_powerfeather.menu.LoopCore.1.build.loop_core=-DARDUINO_RUNNING_CORE=1
+esp32s3_powerfeather.menu.LoopCore.0=Core 0
+esp32s3_powerfeather.menu.LoopCore.0.build.loop_core=-DARDUINO_RUNNING_CORE=0
+
+esp32s3_powerfeather.menu.EventsCore.1=Core 1
+esp32s3_powerfeather.menu.EventsCore.1.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+esp32s3_powerfeather.menu.EventsCore.0=Core 0
+esp32s3_powerfeather.menu.EventsCore.0.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=0
+
+esp32s3_powerfeather.menu.USBMode.default=USB-OTG (TinyUSB)
+esp32s3_powerfeather.menu.USBMode.default.build.usb_mode=0
+esp32s3_powerfeather.menu.USBMode.hwcdc=Hardware CDC and JTAG
+esp32s3_powerfeather.menu.USBMode.hwcdc.build.usb_mode=1
+
+esp32s3_powerfeather.menu.CDCOnBoot.cdc=Enabled
+esp32s3_powerfeather.menu.CDCOnBoot.cdc.build.cdc_on_boot=1
+esp32s3_powerfeather.menu.CDCOnBoot.default=Disabled
+esp32s3_powerfeather.menu.CDCOnBoot.default.build.cdc_on_boot=0
+
+esp32s3_powerfeather.menu.MSCOnBoot.default=Disabled
+esp32s3_powerfeather.menu.MSCOnBoot.default.build.msc_on_boot=0
+esp32s3_powerfeather.menu.MSCOnBoot.msc=Enabled (Requires USB-OTG Mode)
+esp32s3_powerfeather.menu.MSCOnBoot.msc.build.msc_on_boot=1
+
+esp32s3_powerfeather.menu.DFUOnBoot.default=Disabled
+esp32s3_powerfeather.menu.DFUOnBoot.default.build.dfu_on_boot=0
+esp32s3_powerfeather.menu.DFUOnBoot.dfu=Enabled (Requires USB-OTG Mode)
+esp32s3_powerfeather.menu.DFUOnBoot.dfu.build.dfu_on_boot=1
+
+esp32s3_powerfeather.menu.UploadMode.cdc=USB-OTG CDC (TinyUSB)
+esp32s3_powerfeather.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+esp32s3_powerfeather.menu.UploadMode.cdc.upload.wait_for_upload_port=true
+esp32s3_powerfeather.menu.UploadMode.default=UART0 / Hardware CDC
+esp32s3_powerfeather.menu.UploadMode.default.upload.use_1200bps_touch=false
+esp32s3_powerfeather.menu.UploadMode.default.upload.wait_for_upload_port=false
+
+esp32s3_powerfeather.menu.PSRAM.enabled=QSPI PSRAM
+esp32s3_powerfeather.menu.PSRAM.enabled.build.defines=-DBOARD_HAS_PSRAM
+esp32s3_powerfeather.menu.PSRAM.enabled.build.psram_type=qspi
+esp32s3_powerfeather.menu.PSRAM.disabled=Disabled
+esp32s3_powerfeather.menu.PSRAM.disabled.build.defines=
+esp32s3_powerfeather.menu.PSRAM.disabled.build.psram_type=qspi
+
+esp32s3_powerfeather.menu.PartitionScheme.default_8MB=Default (3MB APP/1.5MB SPIFFS)
+esp32s3_powerfeather.menu.PartitionScheme.default_8MB.build.partitions=default_8MB
+esp32s3_powerfeather.menu.PartitionScheme.default_8MB.upload.maximum_size=3342336
```

Comment:
if you have one option only, it's better to remove the menu all together and set the proper values up in the beginning. In your case:
```
esp32s3_powerfeather.build.partitions=default_8MB
esp32s3_powerfeather.upload.maximum_size=3342336
```

Reply:
I've decided to retain these options in the menu, but decided to remove the one in https://github.com/espressif/arduino-esp32/pull/8889#discussion_r1395305488.